frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -2124,6 +2610,63 @@ ConcatenateWith _wire2api_concatenate_with(FlutterRustBridgeExampleSingleBlockTe
   );
 }
 
+CustomError _wire2api_custom_error(dynamic raw) {
+  final access = raw is Map ? raw['dart_object'] : raw;
```

Comment:
Oops this "Map" approach does not sound very neat or fast...Can we improve it a bit?